src/app/components/avatar/avatar.ts

```diff
@@ -18,7 +18,7 @@ import { ChangeDetectionStrategy, Component, EventEmitter, Input, NgModule, Outp
     encapsulation: ViewEncapsulation.None,
     styleUrls: ['./avatar.css'],
     host: {
-        class: 'p-element'
+        class: 'flex p-element'
```

Comment:
I don't know if this is ok because this class will not be applied if the project is not using primeflex.
Maybe it's working in PrimeNG web because they are using it, but in my project and if i'm not wrong,  this will add `flex` and will not do nothing. Check it, please

Reply:
Absolute right here, i've implemented a own p-avatar-host class for it